spec/tom_queue/tom_queue_integration_spec.rb

```diff
@@ -212,7 +212,7 @@ def start!
 
       # Merge the list of received work together sorted by received time,
       # and compare to the source list
-      sink_order = consumers.map { |c| c.work }.flatten.sort.map { |a| a.payload }
+      sink_order = consumers.map(&:work).flatten.sort { |a, b| a[:run_at] <=> b[:run_at] }.map(&:payload)
```

Comment:
Sorting `run_at` will sort based on when the job is scheduled to run. `run_at` is the header on the message indicating the original value of `run_at` passed into the publish method, so no longer actually compares the order that they were processed.

So, no, probably best not to use that as a comparison ;)